modules/installation-mirror-repository.adoc

```diff
@@ -22,12 +22,7 @@ ifdef::openshift-origin[]
 * You have created a pull secret for your mirror repository.
 endif::[]
 
-* If you use self-signed certificates that do not set a Subject Alternative Name, you must precede the `oc` commands in this procedure with `GODEBUG=x509ignoreCN=0`. If you do not set this variable, the `oc` commands will fail with the following error:
-+
-[source,terminal]
-----
-x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0
-----
+* If you use self-signed certificates, you must set a Subject Alternative Name in self-signed certificates.
```

Comment:
A suggested rewording, to avoid the repetition and match verbs with other prerequisites.

"If you use self-signed certificates, you have specified a Subject Alternative Name in the certificates."

Reply:
Done